frontend/src/pages/projects/screens/projects/NotebookStateColumn.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import { NotebookState } from '~/pages/projects/notebook/types';
+import NotebookStatusToggle from '~/pages/projects/notebook/NotebookStatusToggle';
+import { NotebookImageAvailability } from '~/pages/projects/screens/detail/notebooks/const';
+import useNotebookImage from '~/pages/projects/screens/detail/notebooks/useNotebookImage';
+
+type NotebookStateColumnProps = {
+  enablePipelines: boolean;
+  notebookState: NotebookState;
+};
+
+const NotebookStateColumn: React.FC<NotebookStateColumnProps> = ({
```

Comment:
Shouldn't be called `...Column` doesn't represent a column, it represents the toggle for Notebook Status. Maybe something like
```suggestion
const NotebookStateStatus: React.FC<NotebookStateColumnProps> = ({
```